src/items/ItemSkeleton/ItemSkeleton.tsx

```diff
@@ -48,12 +48,14 @@ const ItemSkeleton: React.FC<ItemSkeletonProps> = ({
         />
       );
     }
-    case [
-      ItemType.LOCAL_FILE,
-      ItemType.S3_FILE,
-      ItemType.LINK,
-      ItemType.APP,
-    ].includes(itemType): {
+    case (
+      [
+        ItemType.LOCAL_FILE,
+        ItemType.S3_FILE,
+        ItemType.LINK,
+        ItemType.APP,
+      ] as string[]
+    ).includes(itemType): {
```

Comment:
Why is this typing necessary?

Reply:
Without it, the type of the array is `ItemType[]` and the values of the union type can't be compared because the types do not match.
<img width="814" alt="Capture d’écran 2023-03-02 à 15 05 42" src="https://user-images.githubusercontent.com/39373170/222451032-eeb64dd7-f825-4f04-aed0-05f2414bf762.png">

Reply:
Maybe we can do:
```ts
[
        ItemType.LOCAL_FILE,
        ItemType.S3_FILE,
        ItemType.LINK,
        ItemType.APP,
      ] as `${ItemType}`[]
    ).includes(itemType):
```